lib/MetaCPAN/Server/QuerySanitizer.pm

```diff
@@ -50,10 +50,13 @@ sub _scan_hash_tree {
     my $ref = ref($struct);
     if ( $ref eq 'HASH' ) {
         while ( my ( $k, $v ) = each %$struct ) {
-            if ( $k eq $key ) {
-                MetaCPAN::Server::QuerySanitizer::Error->throw(
-                    message => qq[Parameter "$key" not allowed], );
-            }
+            # Mickey: disabling this check for 'script' key since
+            #         for ES 1.7 I need to use it in the
+            #         function_score syntax
```

Comment:
one of the early hacks. couldn't make the `prefer_shorter_module_names_400` to work as given script name... I translated it to be used through the new `function_score` syntax, but that required a `script` key.
I'll try again to make the original stored script to work and restore the restriction.